source/features/quick-new-issue.tsx

```diff
@@ -23,6 +23,12 @@ function add(dropdownMenu: HTMLElement): void {
 		<li role="presentation" aria-hidden="true" data-view-component="true" className="ActionList-sectionDivider"/>,
 		<li data-targets="action-list.items" role="none" data-view-component="true" className="ActionListItem">
 			<a href={buildRepoURL('issues/new/choose')} tabIndex={-1} role="menuitem" data-view-component="true" className="ActionListContent">
+				<span className="ActionListItem-visual ActionListItem-visual--leading">
+					<svg aria-hidden="true" height="16" viewBox="0 0 16 16" version="1.1" width="16" data-view-component="true" className="octicon octicon-repo-push">
```

Comment:
You can import the icon from the "octagons" package. Refer to its usage in other features

Reply:
It's done.

Reply:
* octicons 😅 damn autocorrect

Reply:
and we have a library named [octagon](https://github.com/Tripwire/octagon) which is thankfully unrelated to icons. 😁